lib/chef-dk/policyfile_lock.rb

```diff
@@ -15,7 +16,7 @@
 # limitations under the License.
 #
 
-require 'digest/sha1'
+require 'digest/sha2'
```

Comment:
Is this going to change people's existing SHA values?  IE, if I have an existing policyfile.lock and I recompute it after this change is accepted, are my SHAs going to change?